static/src/stylesheets/module/identity/_consent.scss

```diff
@@ -35,3 +35,18 @@
         }
     }
 }
+
+.identity-consent-hero {
+    height: $gs-row-height * 4;
+    background-size: cover;
+    border-bottom: 1px solid $garnett-neutral-4;
+    background-position: center;
+    background-repeat: no-repeat;
+    @include mq(tablet) {
+        height: $gs-row-height * 6;
+        background-size: contain;
+    }
+    &.identity-consent-hero--gdpr-oi-campaign {
+        background-image: url(https://s3-eu-west-1.amazonaws.com/identity-public-email-assets/gdpr-oi-hero.svg);
```

Comment:
@jfsoul Is this the proper way to link this asset? I didn't want to throw it into the repo because it's a very one-off thing

Reply:
I don't know where we normally put these kind of SVGs etc, but I don't think this is the right answer.  Can you check with someone on dotcom?  At the very least, we have existing s3 buckets for this kind of thing (behind fastly or imgix).